Gemfile

```diff
@@ -1,6 +1,8 @@
 source "https://rubygems.org"
 
+gem "bcrypt"
```

Comment:
To: @thanhgiong195 bỏ luôn dòng bcrypt đi cũng được, `gem "devise"` sẽ tự động thêm bcrypt vào (yaoming)